kong

```diff
@@ -1 +1 @@
-./submodules/kong
\ No newline at end of file
+../kong
```

Comment:
in your local folder, do you have a `submodules` folder in the `sb-kong-vagrant` repo?

this symlink wouldn't work the way it is, as the repo doesn't have that folder, and the kong submodule is part of the `smartbuildings-public-api` repo

Reply:
it might make more sense for kong to be a submodule of the vagrant repo instead